src/main/java/org/fcrepo/migration/DatastreamVersion.java

```diff
@@ -119,13 +119,4 @@
      * @return  True if this is the first version, false otherwise.
      */
     public boolean isFirstVersionIn(ObjectReference obj);
-
```

Comment:
Even though we're no longer using this method with the project, for other tools that depend on this library,  I wonder if tooling referencing this library might find it useful.

Reply:
Unless you feel strongly otherwise, I would prefer to leave this method where it is.

Reply:
I'll revert that commit.